acvm/src/pwg/mod.rs

```diff
@@ -102,21 +102,31 @@ pub struct ACVM<B: BlackBoxFunctionSolver> {
     /// Note that this doesn't include any opcodes which are waiting on a pending foreign call.
     opcodes: Vec<Opcode>,
 
+    /// The position of the each opcode in the opcodes
+    opcodes_idx: Vec<OpcodeLabel>,
```

Comment:
Can we remove this field and change opcodes to:
```
opcodes: Vec<(Opcode, OpcodeLabel)>,
```

Reply:
Done